docs/model_zoo/index.md

```diff
@@ -4,89 +4,201 @@ MXNet features fast implementations of most state-of-the-art models reported in
-6. ImageNet 11k
-7. [Places2](http://places2.csail.mit.edu/download.html): There are 1.6 million train images from 365 scene categories in the Places365-Standard, which are used to train the Places365 CNNs. There are 50 images per category in the validation set and 900 images per category in the testing set. Compared to the train set of Places365-Standard, the train set of Places365-Challenge has 6.2 million extra images, leading to totally 8 million train images for the Places365 challenge 2016. The validation set and testing set are the same as the Places365-Standard.
-
-
-
-| Model Definition | Dataset | Model Weights | Research Basis | Contributors |
-| --- | --- | --- | --- | --- |
-| CaffeNet | ImageNet | |   [Krizhevsky, 2012](http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks) | @.. |
-| Network in Network (NiN) | CIFAR-10 | |  [Lin et al.., 2014](https://arxiv.org/pdf/1312.4400v3.pdf) | |
-| SqueezeNet | ImageNet | | [Iandola et al.., 2016](https://arxiv.org/pdf/1602.07360v4.pdf) | |
-| VGG16 | ImageNet | | [Simonyan et al.., 2015](https://arxiv.org/pdf/1409.1556v6.pdf) | |
-| VGG19 | ImageNet | | [Simonyan et al.., 2015](https://arxiv.org/pdf/1409.1556v6.pdf) | |
-| Inception v3 w/BatchNorm | ImageNet | | [Szegedy et al.., 2015](https://arxiv.org/pdf/1512.00567.pdf) | |
-| ResidualNet152 | ImageNet | | [He et al.., 2015](https://arxiv.org/pdf/1512.03385v1.pdf) | |
-| Fast-RCNN | PASCAL VOC | | [Girshick, 2015](https://arxiv.org/pdf/1504.08083v2.pdf) | |
-| Faster-RCNN | PASCAL VOC |  | [Ren et al..,2016](https://arxiv.org/pdf/1506.01497v3.pdf) | |
-| Single Shot Detection (SSD) | PASCAL VOC | | [Liu et al.., 2016](https://arxiv.org/pdf/1512.02325v4.pdf) | |
+* [ImageNet](http://image-net.org/): a large corpus of 1 million natural images, divided into 1000 categories.
+* [CIFAR10](https://www.cs.toronto.edu/~kriz/cifar.html): 60,000 natural images (32 x 32 pixels) from 10 categories.
+* [PASCAL_VOC](http://host.robots.ox.ac.uk/pascal/VOC/): A subset of ImageNet images with object bounding boxes.
+* [UCF101](http://crcv.ucf.edu/data/UCF101.php): 13,320 videos from 101 action categories.
+* [Mini-Places2](http://6.869.csail.mit.edu/fa15/project.html): Subset of the Places2 dataset. Includes 100,000 images from 100 scene categories.
+* ImageNet 11k
+* [Places2](http://places2.csail.mit.edu/download.html): There are 1.6 million train images from 365 scene categories in the Places365-Standard, which are used to train the Places365 CNNs. There are 50 images per category in the validation set and 900 images per category in the testing set. Compared to the train set of Places365-Standard, the train set of Places365-Challenge has 6.2 million extra images, leading to totally 8 million train images for the Places365 challenge 2016. The validation set and testing set are the same as the Places365-Standard.
+
+
+
+```eval_rst
```

Comment:
After compilation, the tables of this md file are automatically converted to eval_rst. I'll keep this file as original and push again.